css/style.css

```diff
@@ -0,0 +1,81 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  font-family: Helvetica, Helvetica-Neue, Arial;
+  text-align: center;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Did some research, it appears that IE and safari need vendor prefix's for this. 

Would it be best to do this as a SASS include with these listed:
-ms-transform: translate($xVal,$yVal);
-webkit-transform: translate($xVal,$yVal);
transform: translate($xVal,$yVal);